tests/comparisons/Template/testBakeEdit.ctp

```diff
@@ -1,6 +1,6 @@
-<div class="actions columns large-2 medium-3">
-    <h3><?= __('Actions') ?></h3>
+<nav class="large-3 medium-4 columns" id="actions-sidebar">
     <ul class="side-nav">
+        <li class="heading"><?= __('Actions') ?></li>
```

Comment:
I am not sure having a list of links with the first one being a description of the rest of the list is such a good idea.